src/Server/Database.fs

```diff
@@ -0,0 +1,53 @@
+/// Functions for managing the database.
+module ServerCode.Database
+
+open System.IO
+open Suave
+open Suave.Logging
+open System.Net
+open Suave.Filters
+open Suave.Operators
+open Suave.RequestErrors
+open Microsoft.Azure.WebJobs
+open ServerCode.Storage.AzureTable
+open ServerCode
+
+[<RequireQualifiedAccess>]
+type DatabaseType = 
+    | FileSystem 
+    | Azure of connectionString : AzureConnection
```

Comment:
Should this be `AzureStorage`?

Reply:
For completeness, probably. It's not particularly confusing at the moment as there's no e.g. DocumentDB story just yet :-)